index.html

```diff
@@ -15,12 +15,27 @@
 </head>
 
 <body>
-
-  
-     
+      <nav>
+        <div>
+          <h1>Fun Bus</h1>
+        </div>
+        <div>  
```

Comment:
mmm, youre wanting a `Navigation` bar correct? I'd change the `<div>` to a `<nav>` !